main/main.ino

```diff
@@ -44,6 +64,13 @@ float last_height;
 void loop()
 {
   update_sensors();
+  digitalWrite(PB9, HIGH);
+  delay(800);
+  digitalWrite(PB9, LOW);
+  delay(200);
```

Comment:
Do we want to have that long of a delay in the main loop? In the past we had ~15 control cycles per second, with this less than one.

Reply:
ah no, we only needed that for debugging